plugins/scenario/hu.bme.mit.gamma.scenario.reduction/bin/.gitignore

```diff
@@ -0,0 +1 @@
+/hu/
```

Comment:
I believe every file in the `bin` folder is deleted when rebuilding is carried out.